Gemfile

```diff
@@ -2,49 +2,62 @@
 
 source "https://rubygems.org"
 
-DECIDIM_VERSION = { git: "https://github.com/CodiTramuntana/decidim.git", branch: "release/0.24-stable" }.freeze
+DECIDIM_VERSION = { git: "https://github.com/CodiTramuntana/decidim.git", branch: "release/0.25-stable" }.freeze
 
 ruby RUBY_VERSION
 
 gem "decidim", DECIDIM_VERSION
 gem "decidim-consultations", DECIDIM_VERSION
 gem "decidim-verifications", DECIDIM_VERSION
 
-gem "decidim-action_delegator", git: "https://github.com/coopdevs/decidim-module-action_delegator.git"
-gem "decidim-term_customizer", git: "https://github.com/CodiTramuntana/decidim-module-term_customizer"
-
+gem "decidim-action_delegator", git: "https://github.com/CodiTramuntana/decidim-module-action_delegator.git", branch: "upgrade/decidim-0_25"
 gem "virtus-multiparams"
+gem "omniauth-rails_csrf_protection", "~> 1.0"
+
+gem "decidim-verifications-members_picker", git: "https://github.com/gencat/decidim-verifications-members_picker.git", tag: "0.0.2"
+gem "decidim-term_customizer", git: "https://github.com/mainio/decidim-module-term_customizer", branch: "release/0.25-stable"
 
-gem "bootsnap", "~> 1.3"
 gem "deface"
+gem "wicked_pdf"
+gem "savon", "~> 2.12.0"
 
-gem "daemons"
-gem "decidim-verifications-members_picker", git: "https://github.com/gencat/decidim-verifications-members_picker.git", tag: "0.0.2"
-gem "delayed_job_active_record"
-gem "figaro"
 gem "puma"
-gem "uglifier", "~> 4.1"
-gem "whenever"
+gem "uglifier", ">= 1.3.0"
+gem "webpacker"
 
-gem "faker", "~> 2.14.0"
-gem "wicked_pdf"
+gem "figaro", ">= 1.1.1"
+gem "openssl"
 
-gem "savon", "~> 2.12.0"
+# if deploying to a dedicated server
+gem "daemons"
+gem "delayed_job_active_record"
+gem "whenever"
+# elsif deploying to Heroku
```

Comment:
Maybe we can remove this comment lines.

Reply:
I know that these comments don't apply here. But I'd prefer to keep them to keep the app as similar as possible with the template.